src/components/__tests__/SearchForm-test.tsx

```diff
@@ -60,10 +60,6 @@ describe("SearchForm", () => {
     wrapper.setState({ searchTerm: "a string" });
     wrapper.find("button").simulate("click");
     expect(search.callCount).to.equal(1);
-    setTimeout(
-      () => expect(wrapper.props()["searchCompleted"]).to.be.true,
-      2000
-    );
```

Comment:
The above assertion had no effect, since it ran asynchronously after a 2 second delay, after the test had already completed. It also appears to be an erroneous assertion. There is no reason to expect that the `searchCompleted` prop will change. Asserting that the `search` function is called when the Search button is clicked is sufficient to test this component, so I'm removing this assertion.